lib/components/molecules/Accordian/Accordian.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+/**
+ *
+ * Accordian
+ *
+ */
+import { PureComponent } from 'react';
+import styled from 'styled-components';
+import type { Node } from 'react';
+import styles from './Accordian.style';
+import type { Props, State } from './types';
+
+class Accordian extends PureComponent<Props, State> {
```

Comment:
@DipshiGoyal i would expect you to built this component like this https://codepen.io/amitmojumder/pen/xOKjPW

Reply:
The suggestion has been incorporated.